website/pages/blog/announcing-the-vercel-source-plugin.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Announcing The Vercel Source Plugin
+tag: announcement
+date: 2022/12/14
+description: >-
+  CloudQuery is an open-source, extensible framework that gives you a
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Vercel source plugin.
+author: kemal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Gandi source plugin for CloudQuery.
+
+Vercel is the platform for frontend developers, providing the speed and reliability innovators need to create at the moment of inspiration.
+
+CloudQuery now supports pulling resources from [Vercel](https://vercel.com/) using Vercel's API.
+
+Take a look at our [Configuration](/docs/plugins/sources/vercel/configuration) section to configure required credentials for the plugin.
+
+Below are some real world query examples to get you started.
+
+## Detect domain registrations that will expire soon
```

Comment:
I think Vercel does this for you https://vercel.com/docs/concepts/projects/domains/renew-a-domain, so maybe mention that and also that this provides more flexibility

Reply:
Fixed